<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta content="width=device-width, initial-scale=1.0" name="viewport">

  <title>RECOMB-CCB: Computational Cancer Biology</title>

  <!-- Favicons -->
  <!-- link href="./assets/img/favicon.png" rel="icon"> -->
  <!-- link href="./assets/img/apple-touch-icon.png" rel="apple-touch-icon"> -->

  <!-- Google Fonts -->
  <link rel="preconnect" href="https://fonts.googleapis.com">
  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
  <link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;1,300;1,400;1,600;1,700&family=Amatic+SC:ital,wght@0,300;0,400;0,500;0,600;0,700;1,300;1,400;1,500;1,600;1,700&family=Inter:ital,wght@0,300;0,400;0,500;0,600;0,700;1,300;1,400;1,500;1,600;1,700&display=swap" rel="stylesheet">

  <!-- Vendor CSS Files -->
  <link href="assets/vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
  <link href="assets/vendor/bootstrap-icons/bootstrap-icons.css" rel="stylesheet">
  <link href="assets/vendor/aos/aos.css" rel="stylesheet">
  <link href="assets/vendor/glightbox/css/glightbox.min.css" rel="stylesheet">
  <link href="assets/vendor/swiper/swiper-bundle.min.css" rel="stylesheet">
  <!-- Template Main CSS File -->
  <link href="assets/css/main.css" rel="stylesheet">
 

</head>
<body>
<!-- ======= Header ======= -->
<header id="header" class="header fixed-top d-flex align-items-center">
    <div class="container d-flex align-items-center justify-content-between">
    
      <div>
      <a href="index.html" class="logo d-flex align-items-center me-auto me-lg-0">
        <h1>16th RECOMB-CCB 2024<span>.</span></h1>
      </a>
      <p>April 27-28, 2024, Boston, MA, USA</p>
      </div>
  <nav id="navbar" class="navbar"><ul>
<li><a href="index.html">Home</a></li>
<li><a href="program.html">Program</a></li>
<li><a href="cfp.html">Call for Papers</a></li>
<li><a href="accepted.html">Accepted Submissions</a></li>
<li><a href="https://recomb.org/recomb2024/register.html">Register</a></li>
<li><a href="https://recomb.org/recomb2024/venue_satellites.html">Venue</a></li>
<!--<li><a href="committees.php.html">Committees</a></li>-->
<!--<li><a href="https://recomb.org/recomb2024/">RECOMB</a></li>-->
</ul></nav>

    <i class="mobile-nav-toggle mobile-nav-show bi bi-list"></i>
    <i class="mobile-nav-toggle mobile-nav-hide d-none bi bi-x"></i>
</div>
</header>
<main id="main">
<section class="hero">
<div class="container">
<div class="section-header"><h1>Accepted Submissions</h1></div>


<div class="row">
<div class="col-lg-10">

<h3>Accepted papers</h3>

<ul>
<li>Ethan Kulman, Rui Kuang and Quaid Morris. Orchard: building large cancer phylogenies using stochastic combinatorial search</li>
<li>Ron Zeira, Leon Anavy, Zohar Yakhini, Ehud Rivlin and Daniel Freedman. HIPI: Spatially Resolved Multiplexed Protein Expression Inferred from H&amp;E WSIs</li>
<li>Henri Schmidt and Benjamin Raphael. A regression based approach to phylogenetic reconstruction from multi-sample bulk DNA sequencing of tumors</li>
<li>Nishat Anjum Bristy, Xuecong Fu and Russell Schwartz. Sc-TUSV-ext: Single-cell clonal lineage inference from single nucleotide variants (SNV), copy number alterations (CNA) and structural variants (SV)</li>
<li>Ahmed Shuaibi, Uthsav Chitra and Benjamin Raphael. A latent variable model for evaluating mutual exclusivity and co-occurrence between driver mutations in cancer</li>
<li>Davide Gurnari, Aldo Guzmán-Sáenz, Filippo Utro, Aritra Bose, Saugata Basu and Laxmi Parida. Probing omics data via harmonic persistent homology</li>
<li>Yushu Liu and Luay Nakhleh. On the Performance of Clustering in Large-scale Evolutionary Analyses of Single-Cell DNA Copy Number Data</li>
<li>Chunguang Lai, Arda Durmaz, Doug Brubaker and Gurkan Bebek. CADRE: Utilizing CAncer Drug Response nEtwork for Drug Synergy Prediction</li>
</ul>

<h3>Accepted highlight talks</h3>

<ul>
<li>Marcos Díaz-Gay, Raviteja Vangara, Mark Barnes, Xi Wang, S.M. Ashiqul Islam, Ian Vermes, Stephen Duke, Nithish Bharadhwaj Narasimman, Ting Yang, Zichen Jiang, Sarah Moody, Sergey Senkin, Paul Brennan, Michael R. Stratton and Ludmil B. Alexandrov. Assigning mutational signatures to individual samples and individual somatic mutations with SigProfilerAssignment</li>
<li>H. Robert Frost. Tissue-adjusted pathway analysis of cancer (TPAC): A novel approach for quantifying tumor-specific gene set dysregulation relative to normal tissue</li>
<li>Sukanya Panja, Mihai Truica, Christina Yu, Vamshi Saggurthi, Michael W. Craige, Katie Whitehead, Mayra Tuiche, Aymen Alsaadi, Riddhi Vyas, Shridar Ganesan, Suril Gohel, Frederick Coffman, James Scott Parrott, Songhua Quan, Shantenu Jha, Isaac Kim, Edward Schaeffer, Vishal Kothari, Sarki Abdulkadir and Antonina Mitrofanova. Novel mechanism-centric network-based algorithm identifies molecular programs of treatment resistance in cancer</li>
<li>Mohammadamin Edrisi, Xiru Huang, Huw Alexander Ogilvie and Luay Nakhleh. Accurate integration of single-cell DNA and RNA for analyzing intratumor heterogeneity using MaCroDNA </li>
<li>Mehran Karimzadeh, Christopher Arlidge, Ariana Rostami, Mathieu Lupien, Scott V. Bratman and Michael M. Hoffman. Human papillomavirus integration transforms chromatin to drive oncogenesis</li>
<li>Thomas Watkins, Emma Colliver, Matthew Huska, Tom Kaufmann, Emilia Lim, Cody Duncan, Kerstin Haase, Peter Van Loo, Charles Swanton, Nicholas McGranahan and Roland Schwarz. Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</li>
<li>Manju Anandakrishnan, Karen Ross, Chuming Chen, K.Vijay Shanker and Cathy Wu. KSFinder 2.0, a model for predicting kinases of human phosphorylation sites, and its potential application in inferring dysregulated kinases in case-specific tumors</li>
</ul>

<h3>Accepted lightning talks</h3>

<ul>
<li>Tanveer Ahmad, Mikhail Kolmogorov and Ayse Keskus. Haplotype-specific karyotypes reconstruction and copy number aberrations/profiling from long reads sequencing data</li>
<li>Daria Ostroverkhova, Yiru Sheng and Anna Panchenko. Elucidating rare driver mutations in human cancer</li>
<li>Xiaojun Ma, David Lembersky, Elena Kim, Tullia Bruno, Joseph Testa and Hatice Osmanbeyoglu. Spatial landscape of malignant pleural and peritoneal mesothelioma tumor immune microenvironment</li>
<li>Milad Rafiee Vahid, Andrew Kavran, William Ashworth, Brian Rabe, Anna Lyubetskaya, Eugene Drokhlyansky, Konstantinos Mavrakis, Remco Loos and Jason Dobson. Elucidating transcriptional tumor plasticity in pancreatic adenocarcinoma using pooled single cell RNA CRISPR screens</li>
<li>Michael Geuenich, Barbara Grünwald, Chengxin Yu, Tiak Ju, Amy Zhang, Oumaima Hamza, Gun Ho Jang, Grainne O'Kane, Faiyaz Notta, Steven Gallinger and Kieran Campbell. Identifying HLA mediated antigen presentation deficiency at single cell resolution using transfer learning</li>
<li>Maja-Celine Stöber, Rocio Chamorro Gonzalez, Lotte Brückner, Thomas Conrad, Nadine Wittstruck, Annabell Szymansky, Angelika Eggert, Johannes Schulte, Richard Koche, Anton Henssen, Roland Schwarz and Kerstin Haase. Intercellular extrachromosomal DNA copy number heterogeneity drives cancer cell state diversity</li>
</ul>

<h3>Accepted posters</h3>

<ul>
<li>Thomas Watkins, Emma Colliver, Matthew Huska, Tom Kaufmann, Emilia Lim, Cody Duncan, Kerstin Haase, Peter Van Loo, Charles Swanton, Nicholas McGranahan and Roland Schwarz. Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity</li>
<li>Tanveer Ahmad, Mikhail Kolmogorov and Ayse Keskus. Haplotype-specific karyotypes reconstruction and copy number aberrations/profiling from long reads sequencing data</li>
<li>Brennan Lee, Miguel Fabian Romero Rondon, Kartik Subramanian, Andrea Giovannucci, Ajay Basavanhally, Jason Dobson and Remco Loos. Leveraging Attention to Predict Genomic Instability from Diagnostic H&amp;E Images</li>
<li>Daria Ostroverkhova, Yiru Sheng and Anna Panchenko. Elucidating rare driver mutations in human cancer</li>
<li>Xiaojun Ma, David Lembersky, Elena Kim, Tullia Bruno, Joseph Testa and Hatice Osmanbeyoglu. Spatial landscape of malignant pleural and peritoneal mesothelioma tumor immune microenvironment</li>
<li>Milad Rafiee Vahid, Andrew Kavran, William Ashworth, Brian Rabe, Anna Lyubetskaya, Eugene Drokhlyansky, Konstantinos Mavrakis, Remco Loos and Jason Dobson. Elucidating transcriptional tumor plasticity in pancreatic adenocarcinoma using pooled single cell RNA CRISPR screens</li>
<li>Jia Chirag Parikh and Gaurav Sharma. Development of DNA-Based Aptamers for Targeted Detection of Glioma Tumor Cells in Brain Tissue</li>
<li>Yurui Chen and Louxin Zhang. Advancing Drug Response Predictions with Hierarchical Geometry-Enhanced Deep Learning</li>
<li>Yuju Lee, Edward Chen, Darren Chan, Somi Afiuni, Alina Selega, Miralem Mrkonji, Hartland W. Jackson and Kieran Campbell. Segmentation error aware clustering for highly multiplexed imaging data</li>
<li>Yulong Bai and Andrew Fisher. Integrating Spatial Transcriptomics with Histologic Biomarkers to Enable Forward and Reverse Translational Analyses in Cancer Biology</li>
<li>Michael Geuenich, Barbara Grünwald, Chengxin Yu, Tiak Ju, Amy Zhang, Oumaima Hamza, Gun Ho Jang, Grainne O'Kane, Faiyaz Notta, Steven Gallinger and Kieran Campbell. Identifying HLA mediated antigen presentation deficiency at single cell resolution using transfer learning</li>
<li>Maja-Celine Stöber, Rocio Chamorro Gonzalez, Lotte Brückner, Thomas Conrad, Nadine Wittstruck, Annabell Szymansky, Angelika Eggert, Johannes Schulte, Richard Koche, Anton Henssen, Roland Schwarz and Kerstin Haase. Intercellular extrachromosomal DNA copy number heterogeneity drives cancer cell state diversity</li>
<li>Jiayan Ma, Stephanie Ting, Bartholomew Tam, Timothy Pham, Michael Reich, Jill Mesirov, Pablo Tamayo and William Kim. Deciphering the Functional Roles of Individual Cancer Alleles Across Comprehensive Cancer Genomic Studies</li>
<li>Jia Yao, Ruichen Rong, Tao Wang and Guanghua Xiao. SOAL-YOLO: A High-Speed, High-Accuracy Pathology Image Segmentation Tool Leveraging YOLOv8 and A Novel Overlapping-Aware  Segmentation Loss</li>
<li>Hoi Man Chung and Yuanhua Huang. Interpretable variational encoding of genotypes identifies clonality and lineages in single cells</li>
<li>Ekin Su Erdem and Nurcan Tunçbağ. Network-Modeling Reveals Age-Dependent Patterns in Pan-Cancer Profiles</li>
<li>Asli Dansik, Ece Öztürk and Nurcan Tunçbağ. ECM-Guided Integrative Network Modeling for Patient Stratification</li>
<li>Diya Agarwal and Gaurav Sharma. Computational Simulations of PROTAC as a BRD3 Inhibitor in Neuroblastoma</li>
<li>Anya Goel and Gaurav Sharma. PROFIND: Computational Framework to Generate PROTACs</li>
</ul>

</div>
</div>

</div></section></main><footer id="footer" class="footer"><div class="container">
       <div class="credits">
       &copy; RECOMB-CCB conference 2024<br>
       Designed with help of <a href="https://bootstrapmade.com/">BootstrapMade</a>
<br>Last modified: 4/8/2024.
</div></div></footer> 
 <div id="preloader"></div>

  <!-- Vendor JS Files -->
  <script src="assets/vendor/bootstrap/js/bootstrap.bundle.min.js"></script>
  <script src="assets/vendor/aos/aos.js"></script>
  <script src="assets/vendor/glightbox/js/glightbox.min.js"></script>
  <script src="assets/vendor/purecounter/purecounter_vanilla.js"></script>
  <script src="assets/vendor/swiper/swiper-bundle.min.js"></script>
  <script src="assets/vendor/php-email-form/validate.js"></script>

  <!-- Template Main JS File -->
  <script src="assets/js/main.js"></script>

  </body></html>
